src/GovUk.Education.ExploreEducationStatistics.Admin/Services/ImportService.cs

```diff
@@ -99,8 +99,31 @@ public async Task RemoveImportTableRowIfExists(Guid releaseId, string dataFileNa
 
         public async Task FailImport(Guid releaseId, string dataFileName, IEnumerable<ValidationError> errors)
         {
+            var release = _context.Releases
+                .Where(r => r.Id.Equals(releaseId))
+                .Include(r => r.Publication)
+                .ThenInclude(p => p.Topic)
+                .ThenInclude(t => t.Theme)
+                .FirstOrDefault();
+
+            var importReleaseMessage = JsonConvert.SerializeObject(new ImportMessage()
+            {
+                SubjectId = Guid.Empty,
+                DataFileName = dataFileName,
+                OrigDataFileName = "Unknown",
+                MetaFileName = "Unknown",
```

Comment:
Updated OrigDataFileName.

I've also added MetaFileName as an argument to FailImport, but currently this can return the wrong meta file name due to sometime the ReleaseFileReference row having a null SubjectId. After discussing this with Ben, if I am understanding correctly, this could get fixed as part of EES-1637. And even if it doesn't, it's a failed import and so the metadata filename not being correct isn't really I problem, I think.